tests/switch/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -107,15 +301,17 @@ switch ($expr) {
         break;
 }
 
-switch (
-    $longLongLongLongLongLongLongLongLongLongLongLongLongLongLongLongVariable
-) {
+switch ($longLongLongLongLongLongLongLongLongLongLongLongLongLongLongLongVariable) {
 }
 
-switch (
-    $longLongLongLongLongLongLongVariable &&
-    $longLongLongLongLongLongLongVariable
-) {
+switch ($longLongLongLongLongLongLongVariable &&
+    $longLongLongLongLongLongLongVariable) {
```

Comment:
[playground](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEBnA7gSxmAFgAgAoASAG2gHMAZSmqa2x+uigNQEMAnTdgI1Lj4AZEIA6UfGSYNm0liw7c+A4WJj4Nk8rJ0y98yop784ASnzAAvvhAAaEBAAOMTNFTJQXThHQAFLgjuKOwAbhCYACZ2IOyoMMgAZuykqHD2vJzsYADWcDAAyo5ZmPTIMJwArmkgERBgicmp9iWpnDC+mRQAtuwNKdUAVqgAHgBCmTl5+exdcFQlcH1NDhUwjqsATEvVRZytyCC8fACe2tGO3LAA6pEwuMgAHAAM9hcQqVeZjgcXcK0hi3snDgAEcKphgR12N1ekgkv17KkupgypVqqgShQBABFCoQeDbewwPg3CJ3ZAbImZTCkTEAYQgXR6Byg0EBIAqqQAKnwgvDUpZLEA).

Should we do as `js`?

Reply:
@vjeux it is bug in `prettier` or expected behavior? Don't found issues around this.

Reply:
Looks like we're not applying the same treatment for switch as for `if/while/...`. This is not intended. `switch` is very rarely used in js so it probably never come up in real code.

Reply:
If you can send a fix to prettier that would be very welcome :)

Reply:
@vjeux yep, i will send PR :+1: